moabb/datasets/dreyer2023.py

```diff
@@ -0,0 +1,556 @@
+    A or B.
+    In addition, for each recording the following pieces of information are provided:
+    subject's demographic, personality and cognitive profiles, the OpenViBE experimental
+    instructions and codes, and experimenter's gender.
+
+    A recording contains open and closed eyes baseline recordings and 6 runs of the MI
+    experiments. First 2 runs (acquisition runs) were used to train system and
+    the following 4 runs (training runs) to train the participant. Each run contained
+    40 trials [1]_.
+
+    Each trial was recorded as follows [1]_:
+        - t=0.00s  cross displayed on screen
+        - t=2.00s  acoustic signal announced appearance of a red arrow
+        - t=3.00s  a red arrow appears (subject starts to perform task)
+        - t=4.25s  the red arrow disappears
+        - t=4.25s  the feedback on performance is given in form of a blue bar
+                   with update frequency of 16 Hz
+        - t=8.00s  cross turns off (subject stops to perform task)
+
+    EEG signals [1]_:
+        - recorded with 27 electrodes, namely:
+          Fz, FCz, Cz, CPz, Pz, C1, C3, C5, C2, C4, C6, F4, FC2, FC4, FC6, CP2,
+          CP4, CP6, P4, F3, FC1, FC3, FC5, CP1, CP3, CP5, P3 (10-20 system),
+          referenced to the left earlobe.
+
+    EOG signals [1]_:
+        - recorded with 3 electrodes, namely: EOG1, EOG2, EOG3
+          placed below, above and on the side of one eye.
+
+    EMG signals [1]_:
+        - recorded with 2 electrodes, namely: EMGg, EMGd
+          placed 2.5cm below the skinfold on each wrist.
+
+    Demographic and biosocial information includes:
+        - gender, birth year, laterality
+        - vision, vision assistance
+        - familiarity to cognitive science or neurology, level of education
+        - physical activity, meditation
+        - attentional, neurological, psychiatrics symptoms
+
+    Personality and the cognitive profile [1]_:
+        - evaluated via 5th edition of the 16 Personality Factors (16PF5) test
+        - and mental rotation test
+        - index of learning style
+
+    Pre and post-experiment questionnaires [1]_:
+        - evaluation of pre and post mood, mindfulness and motivational states
+
+    The online OpenViBE BCI classification performance [1]_:
+        - only performance measure used to give the feedback to the participants
+
+
+    References
+    ----------
+
+    .. [1] Pillette, L., Roc, A., N’kaoua, B., & Lotte, F. (2021).
+        Experimenters' influence on mental-imagery based brain-computer interface user training.
+        International Journal of Human-Computer Studies, 149, 102603.
+    .. [2] Benaroch, C., Yamamoto, M. S., Roc, A., Dreyer, P., Jeunet, C., & Lotte, F. (2022).
+        When should MI-BCI feature optimization include prior knowledge, and which one?.
+        Brain-Computer Interfaces, 9(2), 115-128.
+    """
+
+    def __init__(self):
+        super().__init__(subjects=list(range(82, 87)), sub_id="C")
```

Comment:
Here that makes only 5 subjects in dataset C. The docstring says 6, so I guess the total is 88

Reply:
```suggestion
        super().__init__(subjects=list(range(82, 88)), sub_id="C")
```